_posts/2018-05-28-logs-for-the-kovri-pr-meeting-held-on-2018-05-28.md

```diff
@@ -0,0 +1,447 @@
+---
+layout: post
+title: Overview and Logs for the Kovri PR Meeting Held on 2018-05-28
+summary: Discussion of Kovri public relations
+tags: [dev diaries, i2p, crypto]
+author: selsta / fluffypony
+---
+
```

Comment:
Could you add the date here, as on all the posts? `*May 28th, 2018*`

Reply:
I did it like the other log posts, they don’t have a date too.

Reply:
There are, but only for September and October 2017... i looked at the latest kovri one to see this. ATM, it is not a big deal, but maybe we should update the lot to be consistent.

Reply:
It's definitely something to remove where it's present:
![image](https://user-images.githubusercontent.com/37215310/40653901-e318926a-633c-11e8-9887-2c4616eb1136.png)